cockroachdb/templates/service.metrics.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if .Values.metrics.enabled }}
```

Comment:
Is using this service necessary? We already have Prometheus annotations on [this service](https://github.com/cockroachdb/helm-charts/blob/4c89583fbfb9c49f9421079b4e2ca793c7b654ab/cockroachdb/templates/service.discovery.yaml).

Reply:
Using this service - no, but ServiceMonitor requires service. I reused service.discovery. Annotations is a different approach, prometheus-operator using ServiceMonitors